source/Stargate-Model/HTTPBasedRESTfulAPI.class.st

```diff
@@ -68,21 +75,23 @@ HTTPBasedRESTfulAPI >> initializeConfiguredBy: configuration installing: aRESTfu
 
 	teapotServer := Teapot
 		configure: configuration , {(#notFoundHandlerClass -> Tea405AwareNotFoundHandler)}.
-	controllers := aRESTfulControllerCollection
+	controllers := aRESTfulControllerCollection.
+	errorHandlers := OrderedCollection new
 ]
 
 { #category : #actions }
 HTTPBasedRESTfulAPI >> install [
 
 	self
 		configureRoutes;
-		configureHttpClientErrorHandler
+		configureHttpClientErrorHandler;
+		configureErrorHandlers
```

Comment:
Instead of configuring HTTP client error handler and the rest of the handlers separately, HTTP client error handler could be added during initialization. No problem with the order since first added is first applied.